docs/faq.md

```diff
@@ -113,4 +111,8 @@ For the Example project, run the `EarlGreyExampleSwiftTests` target once then fi
 
 For physical device builds, replace `Debug-iphonesimulator` with `Debug-iphoneos`.
 
-You can verify the `EarlGrey.framework` is included in the bundle at `EarlGreyExampleSwiftTests.xctest/Frameworks/EarlGrey.framework`
+**How do I resolve "dyld: could not load inserted library '@executable_path/EarlGrey.framework/EarlGrey' because image not found" error?**
+
+The error means that the dynamic loader is unable to find *EarlGrey.framework* at the specified path: `@executable_path/EarlGrey.framework/EarlGrey`
+
+Verify that *EarlGrey.framework* is embedded in the app under test bundle. Build the **Test Target** and check for EarlGrey.framework in the app under test bundle. For an app named *MyApp*, EarlGrey.framework should be at `MyApp.app/EarlGrey.framework`. If it isn't there, make sure that the **Test Target** has a `Copy to $(TEST_HOST)` script in **Build Phases**. Follow [these instructions](install-and-run.md#final-test-configuration-) on how to configure it. After configuring it, rebuild and check again. If EarlGrey.framework is still not present in the app under test bundle, please [open an issue](https://github.com/google/EarlGrey/issues/new) describing your project setup and the full error in detail.
```

Comment:
I just noticed that if a page is loading when re-directed to the Final Test Configuration, the link is lost since there are a number of images that load above it. Instead of "these instructions", how about - "the Final Test Configuration Setup"